CMakeLists.txt

```diff
@@ -190,14 +190,16 @@ SET(CT_ARCH "i386")
 IF (EMBED_64BIT_CLIENTTOOLS)
   SET(CT_ARCH "x86_64")
 ENDIF ()
-find_file(CLIENTTOOLS_PACKAGE_FILE "hpccsystems-clienttools-community_${version}-${stagever}${CMAKE_SYSTEM_NAME}-${CT_ARCH}.exe" HINTS ${CMAKE_CURRENT_BINARY_DIR}/../HPCC-Platform )
+find_file(CLIENTTOOLS_PACKAGE_FILE "hpccsystems-clienttools-community_${version}-${stagever}${CMAKE_SYSTEM_NAME}-${CT_ARCH}.exe" HINTS ${CMAKE_CURRENT_BINARY_DIR}/../HPCC-Platform ${CMAKE_CURRENT_BINARY_DIR}/../.. ${CMAKE_CURRENT_BINARY_DIR}/.. )
```

Comment:
Added hints that will allow us to find the clienttools package in github actions build-asset workflow